packages/components/inputs/radio-input/src/radio-group.tsx

```diff
@@ -43,8 +40,12 @@ export type TGroupProps = {
     | 16
     | 'scale'
     | 'auto';
-  direction?: keyof typeof directionWrapper;
-  directionProps?: Partial<TStackProps>;
+  direction: 'stack' | 'inline';
+  directionProps: {
+    scale?: TInlineProps['scale'];
+    alignItems?: TInlineProps['alignItems'];
+    justifyContent?: TInlineProps['justifyContent'];
+  };
```

Comment:
Better to be explicit, as otherwise there are other props that do not make sense to allow (e.g. `children`)